Patterns/Turkish/Turkish-Numbers.yaml

```diff
@@ -146,8 +146,8 @@ FractionNounWithArticleRegex: !nestedRegex
   def: (?<=\b)(({AllIntRegex}\s)?(buçuk|çeyrek|yarım))(?=\b)
   references: [ AllIntRegex ]
 FractionPrepositionRegex: !nestedRegex
-  def: (?<!{BaseNumbers.CommonCurrencySymbol}\s*)(?<=\b)(eksi\s)?(?<numerator>({AllIntRegex})|((?<!,)\d+))\s+(bölü)\s+(?<denominator>({AllIntRegex})|(\d+)(?!,))(?=\b)
-  references: [ AllIntRegex, BaseNumbers.CommonCurrencySymbol ]
+  def: (?<=\b)(eksi\s)?(?<numerator>({AllIntRegex})|((?<!,)\d+))\s+(bölü)\s+(?<denominator>({AllIntRegex})|(\d+)(?!,))(?=\b)
```

Comment:
Why were these references removed? Can they be added back?

Reply:
removed by mistake, added back